_includes/header.html

```diff
@@ -21,7 +21,7 @@
         <div class="trigger">
           {% for path in page_paths %}
             {% assign my_page = site.pages | where: "path", path | first %}
-            {% if my_page.title %}
+            {% if my_page.title and !my_page.exclude_from_nav != true %}
```

Comment:
I think the issue is the unary not (`!`), which I understand is not supported in liquid.  I believe this is sufficient:

```
if my_page.title and my_page.exclude_from_nav != true
```

Unfortunately, `== false` isn't available to us here because of a defect with truthiness in liquid/ruby with respect to booleans, nulls, empty string, uninitialized values, and so on...